cli/serve/serve.go

```diff
@@ -192,8 +194,52 @@ func serverMain(args []string, c cli.Config) error {
 	registerHandlers()
 
 	addr := net.JoinHostPort(conf.Address, strconv.Itoa(conf.Port))
-	log.Info("Now listening on ", addr)
-	return http.ListenAndServe(addr, nil)
+
+	endpointCA := conf.EPCAFile
+	endpointKey := conf.EPCAKeyFile
+
+	if endpointCA == "" || endpointKey == "" {
+		log.Info("Now listening on ", addr)
+		return http.ListenAndServe(addr, nil)
+	}
+	var CipherSuites = []uint16{
```

Comment:
The problem isn't to have all the cipher suites supported by the browser, it's to have at least one cipher suite supported by all major browsers (that's the one it will use to negotiate)